### ISSUE TITLE
Inspections - Notes - system should create note based upon PreCon Meeting Attended question

### ISSUE THREAD

**Issue by leeensminger** [kcigeospatial/DelDOT-ES2M#140]
For the first inspection on a Project, where inspection type is not Small Project Construction Diary Inspection or Vegetative Review Form, the system correctly asks "Precon meeting attended?"

The system does not currently automatically create a Note on the inspection to capture the response to this question.

- if Precon meeting attended = Yes, the system should automatically create a Note, where:
  - Date Created = the date on which the inspection was created
  - Date Closed = the date on which the inspection was created (the same as above)
  - Status = "Other"
  - Description = "The precon meeting was attended."

- if Precon meeting attended = No, the system should automatically create a Note, where:
  - Date Created = the date on which the inspection was created
  - Date Closed = the date on which the inspection was created (the same as above)
  - Status = "Other"
  - Description = "The precon meeting was not attended."

**Comment by ehsan67** [comment]
@leeensminger  Where should the "Note" go? can you specify the field?

**Comment by ehsan67** [comment]
@leeensminger  I found the notes, please ignore my previous question

**Comment by ehsan67** [comment]
ready kci

**Comment by impittman** [comment]
@ehsan67 @leeensminger 
The notes are created using the logic provided for capturing Pre-Con Meeting attendance for the first inspection form entered. However, notes are also added to each new inspection for a project stating that a pre-con meeting was not attended. For replication of steps, I did the following:
-Went to a project with no inspections and added the very first inspection to the project, an IFR.
-Selected Pre-Con = Yes and got this note, as expected.
![precon_good](https://user-images.githubusercontent.com/16418393/39193498-3bc5e13e-47cb-11e8-9107-e7b417db4293.png)

-I added a second inspection to the same project and a note regarding Pre-Con meetings was also created for the second inspection of the project (see below). From my understanding, this should not happen on additional inspections that are added.
![precon_bad](https://user-images.githubusercontent.com/16418393/39193539-55320ed6-47cb-11e8-8756-764364a533d9.png)
